components/layouts/PageFooter/index.vue

```diff
@@ -2,32 +2,32 @@
   <footer class="the-page-footer">
```

Comment:
Now I'm confused, I thought this would fall under your definition of:
> A component that is used across all the web, generally included as part of the main layout

Also, you agreed in prefixing this very component (discussed here #660 and implemented here #842).

In any case, please remember to open the discussion to clarify the rule and, if sticking with the new name, rename the root class to be `page-footer`.

Reply:
When I agreed to the discussion we only had one instance of the footer. Now we have two and we create them by using props, so according to the [style guide](https://vuejs.org/v2/style-guide/#Single-instance-component-names-strongly-recommended), it's no longer a single-instance component and we should remove the `The` prefix

I will remove the `the-` from the classes because I forgot to do it

Reply:
Got it. :+1: